[iOS] Scripts for setting up benchmark projects

xta0 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#28469 [iOS] Scripts for setting up benchmark projects**

### Summary

As described [here](https://github.com/pytorch/pytorch/pull/28405), This PR is the second one that contains scripts for setting up the benchmark projects.

### Test Plan

Don't break CI jobs unless they are flaky.

Differential Revision: [D18097248](https://our.internmc.facebook.com/intern/diff/D18097248)

xta0 committed:
[iOS] Scripts for setting up benchmark projects

xta0 committed:
Update on "[iOS] Scripts for setting up benchmark projects"


### Summary

As described [here](https://github.com/pytorch/pytorch/pull/28405), This PR is the second one that contains scripts for setting up the benchmark projects.

### Test Plan

Don't break CI jobs unless they are flaky.

xta0 committed:
Update on "[iOS] Scripts for setting up benchmark projects"


### Summary

As described [here](https://github.com/pytorch/pytorch/pull/28405), This PR is the second one that contains scripts for setting up the benchmark projects.

### Test Plan

Don't break CI jobs unless they are flaky.

ljk53 commented on ios/TestApp/benchmark/setup.rb:
```diff
@@ -0,0 +1,60 @@
+if not File.exist?(model_path_src)
+   raise "no model can be found!"
+end
+FileUtils.cp(model_path_src, model_path_dst)
```
why do we need copy model.pt around? seems a bit confusing to have the same file at both "./" and "../".
can we directly use "TestApp/benchmark/model.pt" from the project? or make "trace_model.py" to save to "TestApp/model.pt" directly?

xta0 replied:
Let's keep the model in `TestApp/benchmark`

xta0 committed:
Update on "[iOS] Scripts for setting up benchmark projects"


### Summary

As described [here](https://github.com/pytorch/pytorch/pull/28405), This PR is the second one that contains scripts for setting up the benchmark projects.

### Test Plan

Don't break CI jobs unless they are flaky.
